---
layout: page
title: projects
permalink: /projects/
description:
nav: true
nav_order: 2
display_categories: [work, fun]
horizontal: false
---

Design principles of gene regulation
======
Gene regulation is a deeply complex process that involves interplay between cellular signaling pathways and the epigenetic landscape. Understanding the fundamental biology that governs this process can tell us where we came from and how we came to be, both as individuals and as a species. Perturbing and engineering gene regulation, on the other hand, can allow us to control our fates: we can shift cell behavior and even mitigate or prevent disease. **To this end, my work broadly focuses on teasing apart the organizational rules that govern gene regulation**. I focus on chromatin structure, which forms an interface between environmental signals and the transcriptional responses to these signals. I use a diverse set of experimental approaches, including (but not limited to) live cell imaging, cellular optogenetics, single-molecule and single-cell transcriptomics, and epigenomic profiling.

<div class="row">
    <img src="https://sidujena.github.io/assets/img/chromatinfig.png" alt="image" width="50%" height="auto">
    <img src="https://sidujena.github.io/assets/img/chromdata.png" alt="image" width="50%" height="auto">
</div>
<br>
<br>

Previous work
======
In my PhD, I primarily focused on the recently discovered phenomenon of Erk signaling dynamics, where the Erk pathway activates and deactivates in a pulsatile fashion. I asked two main questions in my work: **1. How does such rapid pulsatile behavior feed forward to the level of Erk-responsive gene activation?** and **2. How can we infer the sources of signaling pulses in a population of cells that display both cell-autonomous and cell-to-cell coupling behavior in the same pathway?** My work uncovered a set of amplitude, duration, and context-dependent rules for fractional transcriptional responses to transient Erk signaling [(Jena et. al. 2021)](https://www.biorxiv.org/content/10.1101/2021.04.30.442166v1). In addition, my collaborators and I established a set of models and accompanying machine learning tools for inferring signaling behavior from live-cell imaging [(Verma\*, Jena\* et. al. 2021)](https://www.pnas.org/doi/10.1073/pnas.2026123118). My work connecting time-dependent signaling to gene regulation has also extended to developmental, whole-animal contexts, where I dissected the contributing enhancer elements that allow for rapid interpretation of an upstream transcription factor gradient in the fly embryo [(Keller\*, Jena\* et. al. 2020)](https://www.pnas.org/doi/abs/10.1073/pnas.1917040117).

My undergrad thesis, completed under the mentorship of Martin Karplus, was focused on using molecular dynamics to simulate the effect of membrane lipid composition on the behavior of embedded proteins. I also studied models of protocellular membrane evolution with Jack Szostak [(Lin\*, Kamat\* et. al. 2018)](https://onlinelibrary.wiley.com/doi/10.1002/smll.201704077).

Non-research Interests
======
I like spending time outside, and am always on the lookout for a good coffee or sandwich. I also enjoy reading/watching/looking at/listening to/discussing literature, films, art, and music. 
